README.md

```diff
@@ -518,14 +518,12 @@ The `apollo` service has the following public API:
         return {};
       }
       return new Promise(success => {
-        this.get('session').authorize(
-          'authorizer:oauth2',
-          (headerName, headerContent) => {
-            let headers = {};
-            headers[headerName] = headerContent;
-            success({ headers });
-          }
-        );
+        let headers = {};
+        if (this.get('session.isAuthenticated')) {
```

Comment:
No need for this check as we return early if not authenticated. See line 517.

Reply:
you are correct, fixed